eos/b-decays/b-to-psd-nu-nu.cc

```diff
@@ -52,6 +52,10 @@ namespace eos
 
         UsedParameter m_P;
 
+        UsedParameter m_b;
```

Comment:
You don't need these and should not use them :-)

You need the b and s quark masses at the scale of your WET. You can obtain it from your model using ``model->m_b_msbar(mu)`` and ``model->m_s_msbar(mu)``.

You need to add a new ``UsedParameter`` for the scale though. The name of the parameter should be ``sbnunu::mu``, if it has not yet been added to the list of parameters.